pyincore/client.py

```diff
@@ -16,6 +18,63 @@
 logger = pyglobals.LOGGER
 
 
+def edit_hash_entry(data):
+    """
+    Helper function to add/remove the hashed url to service.json file to keep track of
+    the data coming from various repositories.
+    data = {
+            mode (str): String value indicating operation mode
+            hashed_url (optional str): String value of the Hashed Service URL
+            service_url (optional str): Actual Service URL value
+        }
```

Comment:
instead of pass data as a dictionary (which can be any form), could you just make "mode", "hashed_url" and "service_url" three separate input parameters?

Reply:
Okay, got it. I will do it.